QUEST.tsv

```diff
@@ -1408,96 +1408,96 @@ QUEST_20150317_001407	Sincerity of Moze
 QUEST_20150317_001408	Play with Fire!
 QUEST_20150317_001409	Fight Poison with Poison
 QUEST_20150317_001410	In the Name of Faith
-QUEST_20150317_001411	Bramble cheats
+QUEST_20150317_001411	Bramble Cheats
 QUEST_20150317_001412	Activate Obelisk
 QUEST_20150317_001413	Andale Priest's Favor
 QUEST_20150317_001414	Making Languid Flower Bomb
 QUEST_20150317_001415	Bowl for Soul
 QUEST_20150317_001416	Mercy and Salvation
 QUEST_20150317_001417	Research of the Priest
-QUEST_20150317_001418	Hardwork of the Priest
+QUEST_20150317_001418	Hard Work of the Priest
 QUEST_20150317_001419	Goodbye, My Friend
 QUEST_20150317_001420	Adaptation to Circumstances
 QUEST_20150317_001421	Last Mission of the Bishop
 QUEST_20150317_001422	Research Data
 QUEST_20150317_001423	Pharmacist's Favor
-QUEST_20150323_001424	Now go to Gintara Road if Sirdgela Forest. {nl}Spray this potion on the matsum dolls I placed and tell me the results. 
-QUEST_20150323_001425	Not long ago, four scouts were sent to Sirdgela Forest.. but all of them went missing. {nl}Perhaps they took the Thorn Forest too lightly.. Please find their whereabouts.
-QUEST_20150323_001426	I have no strength but I write this for someone who would pass by this area. {nl}Naktis.. It's a funny thing to wish for a curse from the Monarch of Curse. {nl}
-QUEST_20150323_001427	Naktis spread cursed Tree Root Crystals in the area. {nl}To stop the curse, all those Tree Root Crystals must be destroyed. {nl}
-QUEST_20150323_001428	Maybe I will soon lose my breath. {nl}Pilgrims. May the blessings of Goddess be with you!
-QUEST_20150323_001429	Please be sure to find the scripture! {nl}One of soul of pilgrim entrapped in here must have it. 
-QUEST_20150323_001430	All pilgrims should offer different amounts.
+xQUEST_20150323_001424	Now go to Gintara Road in Sirdgela Forest. {nl}Spray this potion on the matsum dolls I placed there and tell me what happens. 
+xQUEST_20150323_001425	Four scouts were sent to Sirdgela Forest not too long ago but all of them went missing. {nl}Maybe they took the Thorn Forest too lightly… Please find their whereabouts.
+xQUEST_20150323_001426	I have no strength but I write this for anyone may pass by this area. {nl}Naktis… It's a funny thing to wish a curse on the Monarch of Curses.
+QUEST_20150323_001427	Naktis spread cursed Tree Root Crystals in the area. {nl}To stop the curse, they must be destroyed.
```

Comment:
Might be more clear for the player if you said, "All the Crystals must be destroyed"